relayer/chains/substrate/event_parser.go

```diff
@@ -63,6 +68,7 @@ func (scp *SubstrateChainProcessor) ibcMessagesFromEvents(
 	// add all of accumulated packets to messages
 	for _, pkt := range packetAccumulator {
 		messages = append(messages, ibcMessage{
+			// TODO (Question): will the packet type always be a receive packet?
```

Comment:
I wouldn't think so but I don't think I fully understand the packet accumulator abstraction. I think @agouin would be the most insightful here.

Reply:
@jtieri Let it be as is, We need to modify this part when the code is fully merged with ibctest and running servers